packages/libircclient/patch-fix-warnings

```diff
@@ -0,0 +1,99 @@
+--- src/colors.c
```

Comment:
Instead of making these changes, it's also perfectly fine to add `unsafe_string_sources` to the `BUILD` file. Just grep other packages for examples. This puts a `#define` at the top of the source file that makes cloudlibc expose the unsafe string functions.

I agree it's better if upstream moves away from the unsafe string functions, but I'd rather not make such changes locally. If we do something wrong with porting these functions over to use the safe versions, we could in fact create security bugs that weren't there before...